.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,17 @@ Fixes #
 
 **Special notes for your reviewer**:
 
+**Checklist**
+
+This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
+Approvers are expected to review this list.
+
+- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
+- [ ] Maintenance: The [Boy Scout Rule](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html) was considered and applied if needed
+- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least one e2e test
```

Comment:
Not every bugfix requires an e2e test, actually most don't, we prefer unit tests :)

Reply:
A failing e2e test that proves the bug guards against regressions. However, I agree on that we prefer unit tests, so I'd say let's write many unit tests but few e2e tests.